src/lib/x509/x509_ext.h

```diff
@@ -500,6 +500,66 @@ class OCSP_NoCheck final : public Certificate_Extension {
       void decode_inner(const std::vector<uint8_t>&) override;
 };
 
+/**
+* TNAuthList extension
+*
+* RFC8226 Secure Telephone Identity Credentials
+*   https://www.rfc-editor.org/rfc/rfc8226#section-9
+*/
+class BOTAN_PUBLIC_API(3, 6) TNAuthList final : public Certificate_Extension {
```

Comment:
```suggestion
class BOTAN_PUBLIC_API(3, 5) TNAuthList final : public Certificate_Extension {
```

... Botan 3.5.0 is coming soon. Let's be optimistic. :)